### ISSUE TITLE
Freak Boy - Unhandler R4300i opcode

### ISSUE THREAD

**Issue by oddMLan** [project64/project64#1830]
![image](https://user-images.githubusercontent.com/7278372/85951268-ed4eb980-b916-11ea-8442-4ed69000b38f.png)

Need to disable Advanced Block Linking to get it to boot
Plugins - All default, GLideN64 as video plugin

How to reproduce: Press B to shoot. Savestate:
[Freak Boy (Unrelease.pj.zip](https://github.com/project64/project64/files/4842489/Freak.Boy.Unrelease.pj.zip)

**Comment by project64** [comment]
break, is a fatal error ...

does it work work with Advanced Block linking, disabled, or just boots.

**Comment by oddMLan** [comment]
It works fine, enters in-game with ABL. Without it, it doesn't even boot

**Comment by project64** [comment]
sounds like a bug in ABL then

**Comment by oddMLan** [comment]
Yeah, it is also a bug in ABL if it makes it crash. However if you disable ABL to bypass the error then you get that error.

**Comment by project64** [comment]
does it run on interpter?

**Comment by oddMLan** [comment]
Yes, it runs fine in interpreter. with the savestate I provided, the missile doesn't trigger the error. however the textures are all glitched

**Comment by olivieryuyu** [comment]
disable 32 bits and all works fine

**Comment by oddMLan** [comment]
> disable 32 bits and all works fine

32 bit engine fixes the glitchy lighting in Freak Boy
Recompiler still crashes with that break error when firing a missile
